README.md

```diff
@@ -187,3 +187,17 @@ Simply add a `release` property to your `package.json` file. For example:
   // ... props
 }
 ```
+
+## Rollup
+
+By default all `dependencies` and `peerDependencies` declared in your `package.json` will be treated as external deps during `sl-scripts bundle`. If you would like to always bundle a dep, list those deps in your `package.json` file like so:
+
+```json
+{
+  "name": "your-package",
+  "version": "0.0.0",
+  "rollup": {
```

Comment:
I hope Rollup never decides to support some configuration via package.json 🙈

Reply:
lol yeah - i mean i could pick a different key? what do you think, should we?

Reply:
`sl_rollup`? I don't really know, something with Stoplight would be probably best.
Or maybe `@stoplight/scripts`.

Reply:
Meh, we'll be using this very very rarely - let's just go with it for now, can always update down the road in the (pretty unlikely) event that there's a conflict in future.

Reply:
Yeah, fair enough.